correios/renderers/pdf.py

```diff
@@ -135,6 +135,9 @@ def draw(self):
         # extra services (first three)
         first_row = self.y2 - (40 * mm) - 10  # font-size=10pt
         for extra_service in self.shipping_label.extra_services:
+            if not extra_service.display_on_label:
+                continue
```

Comment:
Seems there is no test passing here.

Reply:
Done the test. Thanks for the tips @cacarrara